packages/main/src/RPA/PDF.py

```diff
@@ -1157,12 +1214,24 @@ def _is_match_on_vertical(self, direction, item, strict, regexp):
                 return item
         return None
 
+    def _is_match_in_box(self, item):
```

Comment:
`RPA.core.geometry` has primitives from points and regions with common helper methods. not necessary maybe to refactor here, but as a future note.